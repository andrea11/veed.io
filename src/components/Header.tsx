import Link from "next/link";
import Image from "next/image";

export default function Header() {
    return (
        <header className="bg-white">
            <nav className="mx-auto flex max-w-7xl items-center justify-between p-6 lg:px-8" aria-label="Global">
                <div className="flex lg:flex-1">
                    <Link href="/" className="-m-1.5 p-1.5">
                        <span className="sr-only">Your Company</span>
                        <Image src={"/github.svg"} width={32} height={32} alt={"Logo"}></Image>
                    </Link>
                </div>
                <div className="hidden lg:flex lg:flex-1">
                    <Link href="/favourites" className="text-sm font-semibold leading-6 text-gray-900">
                        Favourite repositories <span aria-hidden="true"></span>
                    </Link>
                </div>
                <div className="hidden lg:flex lg:flex-1">
                    <Link href="/repositories" className="text-sm font-semibold leading-6 text-gray-900">
                        Discover trending repositories <span aria-hidden="true"></span>
                    </Link>
                </div>
            </nav>
        </header>
    )
}