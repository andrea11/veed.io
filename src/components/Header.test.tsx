import Header from "@/components/Header";
import {render} from "@testing-library/react";

describe("Header", () => {
    it("renders correctly", () => {
        const { container } = render(<Header/>)
        expect(container).toMatchSnapshot()
    })
})