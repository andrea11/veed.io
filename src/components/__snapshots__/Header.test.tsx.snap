// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div>
  <header
    class="bg-white"
  >
    <nav
      aria-label="Global"
      class="mx-auto flex max-w-7xl items-center justify-between p-6 lg:px-8"
    >
      <div
        class="flex lg:flex-1"
      >
        <a
          class="-m-1.5 p-1.5"
          href="/"
        >
          <span
            class="sr-only"
          >
            Your Company
          </span>
          <img
            alt="Logo"
            data-nimg="1"
            decoding="async"
            height="32"
            loading="lazy"
            src="/github.svg"
            style="color: transparent;"
            width="32"
          />
        </a>
      </div>
      <div
        class="hidden lg:flex lg:flex-1"
      >
        <a
          class="text-sm font-semibold leading-6 text-gray-900"
          href="/favourites"
        >
          Favourite repositories 
          <span
            aria-hidden="true"
          />
        </a>
      </div>
      <div
        class="hidden lg:flex lg:flex-1"
      >
        <a
          class="text-sm font-semibold leading-6 text-gray-900"
          href="/repositories"
        >
          Discover trending repositories 
          <span
            aria-hidden="true"
          />
        </a>
      </div>
    </nav>
  </header>
</div>
`;
